### DISCUSSION TITLE
Don't show new version notifications for patch releases

### DISCUSSION THREAD

**Discussion by tanner37** [immich-app/immich#13005] [Feature Request]
### The bug

Please stop displaying new version notifications for patch releases (ie. v1.116.X).

Too many get sent and it's annoying dismissing the popup every time. **There were already two today.**

Minor (v1.X.0) and major (vX.0.0) release notifications are of course welcome, because they often include a new feature.

Displaying too many notifications risks users turning them off entirely, similar to [alert fatigue](https://en.wikipedia.org/wiki/Alarm_fatigue). Thanks!

### The OS that Immich Server is running on

Debian GNU/Linux 11

### Version of Immich Server

v1.116.0

### Version of Immich Mobile App

N/A

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

N/A

**Comment by danieldietzler** [comment]
Note that we currently don't do semver. When we switch to semver (with immich getting stable) we can look into such things, yeah

**Reply by tanner37** [reply]
Okay, what would you call the v1.1.X class of releases?

**Reply by danieldietzler** [reply]
We do call them "patches" :D

**Comment by mmomjian** [comment]
Also, these patches usually contain important bug fixes that can’t wait until the next big release, so not notifying of them sort of defeats the point.

**Comment by bo0tzz** [comment]
If you prefer, you can disable the version check [in the admin settings](https://my.immich.app/admin/system-settings?isOpen=version-check) and get notifications another way, for example by subscribing to releases on github.

**Reply by goooooouwa** [reply]
I disabled the version check. However I'm still getting new version notifications everytime I open immich.

My current Server Version: v1.116.2
Latest Version: v1.118.2
